test/run_test.vim

```diff
@@ -1,4 +1,16 @@
 "-----------------------------------------------------------------------
+" Global configurations
+"-----------------------------------------------------------------------
+" Configure custom syntax
+let g:verilog_syntax_custom = {
```

Comment:
Do you mean having the users directly modify the `g:verilog_syntax` variable?

For now I wanted to keep that global variable solely under our control. Obviously, I'm not enforcing it, but if something breaks because someone messes with it I can always say that doing so is not supported. Also, custom syntax definitions for now have higher priority. I'm not happy with the solutions I thought of on how to include priority configuration in these variables, so I ended with this simple approach for now.

Obviously, I'm open to suggestions on all this! So feel free to propose different approaches :)